src/Controller/CaseController.php

```diff
@@ -596,10 +596,17 @@ public function assignCaseworker(CaseEntity $case, UserRepository $userRepositor
 
         if ($assignForm->isSubmitted() && $assignForm->isValid()) {
             $this->getDoctrine()->getManager()->flush();
-            $this->addFlash('success', new TranslatableMessage('Case worker {name} assigned to case {case_number}', [
-                'name' => $case->getAssignedTo()->getName(),
-                'case_number' => $case->getCaseNumber(),
-            ], 'case'));
+
+            if ($case->getAssignedTo()) {
+                $this->addFlash('success', new TranslatableMessage('Case worker {name} assigned to case {case_number}', [
+                    'name' => $case->getAssignedTo()->getName(),
+                    'case_number' => $case->getCaseNumber(),
+                ], 'case'));
+            } else {
+                $this->addFlash('success', new TranslatableMessage('Removed assigned case worker from case {case_number}', [
```

Comment:
It would be nice to tell which case worker has been removed from the case.

Reply:
Added more telling flash-messages.